dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/communityManager/accessRequests/AccessToActiveMatchingDatasetTest.kt

```diff
@@ -0,0 +1,187 @@
+package org.dataland.e2etests.tests.communityManager.accessRequests
+
+import org.dataland.communitymanager.openApiClient.api.RequestControllerApi
+import org.dataland.communitymanager.openApiClient.model.AccessStatus
+import org.dataland.communitymanager.openApiClient.model.CompanyRole
+import org.dataland.communitymanager.openApiClient.model.RequestStatus
+import org.dataland.datalandbackend.openApiClient.model.CompanyAssociatedDataVsmeData
+import org.dataland.datalandbackend.openApiClient.model.VsmeData
+import org.dataland.datalandbackendutils.utils.sha256
+import org.dataland.e2etests.BASE_PATH_TO_COMMUNITY_MANAGER
+import org.dataland.e2etests.auth.JwtAuthenticationHelper
+import org.dataland.e2etests.auth.TechnicalUser
+import org.dataland.e2etests.tests.frameworks.Vsme.FileInfos
+import org.dataland.e2etests.utils.ApiAccessor
+import org.dataland.e2etests.utils.FrameworkTestDataProvider
+import org.dataland.e2etests.utils.VsmeUtils
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import java.io.File
+import java.util.*
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class AccessToActiveMatchingDatasetTest {
+
+    val apiAccessor = ApiAccessor()
+    val vsmeUtils = VsmeUtils()
+    private val requestControllerApi = RequestControllerApi(BASE_PATH_TO_COMMUNITY_MANAGER)
+    val jwtHelper = JwtAuthenticationHelper()
+
+    private val testVsmeData = FrameworkTestDataProvider(VsmeData::class.java).getTData(1).first()
+    private lateinit var dummyFileAlpha: File
+    private val fileNameAlpha = "Report-Alpha"
+    private lateinit var hashAlpha: String
+
+    private val timeSleep: Long = 3000
+    lateinit var companyId: String
+
+    @AfterAll
+    fun deleteDummyFiles() {
+        assertTrue(dummyFileAlpha.delete())
+    }
+
+    @Test
+    fun privateFrameworkHasAccess() {
```

Comment:
usually, for tests, we use names like
```
fun `check that user is granted access to private dataset`() {
 // body here
}
```
in any case, this method name is not explanatory enough as to what is tested

Reply:
Added more descriptive titles to the tests